narwhals/dataframe.py

```diff
@@ -195,17 +195,22 @@ class DataFrame(BaseFrame):
     Examples:
         Constructing a DataFrame from a dictionary:
 
+        >>> import pandas as pd
         >>> import polars as pl
         >>> import narwhals as nw
         >>> data = {"a": [1, 2], "b": [3, 4]}
         >>> df_pl = pl.DataFrame(data)
-        >>> df = nw.DataFrame(df_pl)
-        >>> df
-        ┌───────────────────────────────────────────────┐
-        | Narwhals DataFrame                            |
-        | Use `narwhals.to_native` to see native output |
-        └───────────────────────────────────────────────┘
-        >>> nw.to_native(df)
+        >>> df_pd = pd.DataFrame(data)
+
+        Define a library-agnostic function:
+
+        >>> def func(df_any):
+        ...     df = nw.from_native(df_any)
+        ...     return nw.to_native(df)
+
+        Show that you can pass either pandas or Polars to the function `func`:
```

Comment:
```suggestion
        You can pass either pandas or Polars to the function `func`:
```

Reply:
Noted Marco. Will do fix that right away